images/diagnostics/Dockerfile

```diff
@@ -0,0 +1,9 @@
+#
+# OpenShift diagnostics image
+# Used by network diagnostics (oadm diagnostics NetworkCheck)
+#
+# The standard name for this image is openshift/diagnostics-deployer
+
+FROM docker.io/busybox
```

Comment:
I'm good with using the rhel7 base image directly. No need to make the base origin/ose/specific in any way. But I am not ok with using some random image from dockerhub (busybox) that we do not control and/or provide updates and support for.